README.adoc

```diff
@@ -98,61 +98,41 @@ export DOCKER_API_VERSION=1.20
 the time of `vagrant up` itself. Every run of
 `vagrant service-manager env docker` checks for the validity of the
 certificates on the host machine by matching the certificates inside the
-box. If the certificates on the host machine are invalid, this command
+VM. If the certificates on the host machine are invalid, this command
 will also re-download the certificates onto the host machine.
 
 === Available commands
 
-The following section lists the available commands for the plugin and
-their explanation:
+The following section lists the high level commands available for the plugin,
+which enable you to:
 
-1.  `vagrant service-manager [command] [--help | -h]`
+- set up your environment variables and get the TLS certificates to secure the
+Docker communication channel;
+- identify the routable ip network and version of your VM;
```

Comment:
"routable ip network"!? 'ip network' does not make sense imo. "routable IP address as well as version of your VM"